gp-templates/translations.php

```diff
@@ -457,6 +457,17 @@
 	endif;
 ?>
 </table>
+<?php
+$def_vars = get_defined_vars();
```

Comment:
I was looking for the `get_defined_vars()` usage in the filters and actions and I found that this approach was used [here](https://github.com/GlotPress/GlotPress/blob/aaa034e8d188010a2a1ad5abef96c06d65377aaa/locales/locales.php#L2813-L2825), instead of using the `get_defined_vars()` function directly in the filter or action. I did this to maintain the consistency, but, of course, it could be added directly to the `apply_filters()`.

Reply:
Ok, then I'd vote for removing that variable because if there are subsequent calls to `get_defined_vars()` it will contain a `def_vars` entry with the duplicated values.